capabilities/communication/email-templates/maps/sendgrid.suma

```diff
@@ -26,7 +26,9 @@ map ListTemplates {
 }
 
 map GetTemplateContent {
-  http GET "/v3/templates/{input.id}" {
+  inputId = `${input.id}`
+
+  http GET "/v3/templates/{inputId}" {
```

Comment:
I already started working on fixing this lint rule in parsers PR [#79](https://github.com/superfaceai/parser/pull/79). Do we want to include it in 1.0.0 station?

Reply:
I'd say this is legit. It just don't rely on OneSDK type casting.